src/App.js

```diff
@@ -1,19 +1,43 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState } from 'react';
 
 const App = () => {
+  const [chatData, setChatData] = useState(chatMessages);
+
+  const updateChatData = (id) => {
```

Comment:
Consider renaming this method to be a little more specific about what is being updated, maybe a name like `toggleLike` or `updateLikeStatus`